File: CSharp/DependencyBreakingTests/B/MarketingCampaignTest.cs
using Org.Codecop.Dependencies.B;
using Xunit;

namespace Org.Codecop.Dependencies.B.Tests
{
    public class MarketingCampaignTest
    {
        [Fact]
        public void Test2()
        {
            var campaign = new MarketingCampaign();

            var isCrazySalesDay = campaign.IsCrazySalesDay();

            Assert.False(isCrazySalesDay);
        }
    }
}

File: PHP/test/B/MarketingCampaignTest.php
<?php

namespace Org\Codecop\Dependencies\Tests\B;

use PHPUnit\Framework\TestCase;
use Org\Codecop\Dependencies\B\MarketingCampaign;

class MarketingCampaignTest extends TestCase {

    public function test2() {
        $campaign = new MarketingCampaign();

        $isCrazySalesDay = $campaign->isCrazySalesDay();

        $this->assertFalse($isCrazySalesDay);
    }
}
